support-frontend/assets/components/spinners/animatedDots.tsx

```diff
@@ -1,26 +1,71 @@
 // ----- Imports ----- //
+import { css } from '@emotion/core';
+import { neutral } from '@guardian/src-foundations';
 import React from 'react';
-import { classNameWithModifiers } from 'helpers/utilities/utilities';
-import './animatedDots.scss';
+
 // ----- Component ----- //
+
 type PropTypes = {
 	appearance: 'light' | 'medium' | 'dark';
-	modifierClasses: Array<string | null | undefined>;
 };
-export default function AnimatedDots(props: PropTypes) {
+
+export default function AnimatedDots({ appearance }: PropTypes): JSX.Element {
 	return (
-		<div
-			className={classNameWithModifiers('component-animated-dots', [
-				props.appearance,
-				...props.modifierClasses,
-			])}
-		>
-			<div className="component-animated-dots__bounce1" />
-			<div className="component-animated-dots__bounce2" />
-			<div className="component-animated-dots__bounce3" />
+		<div css={styles.container}>
+			<div css={styles.bounce} data-appearance={appearance} data-bounce="1" />
+			<div css={styles.bounce} data-appearance={appearance} data-bounce="2" />
+			<div css={styles.bounce} data-appearance={appearance} data-bounce="3" />
 		</div>
 	);
 }
-AnimatedDots.defaultProps = {
-	modifierClasses: [],
+
+// ---- Styles ---- //
+
+const styles = {
+	container: css`
+		margin: 10px auto 0;
+		width: 70px;
+		text-align: center;
+	`,
+	bounce: css`
+		width: 18px;
+		height: 18px;
+		border-radius: 100%;
+		display: inline-block;
+		animation: sk-bouncedelay 1.4s infinite ease-in-out both;
+
+		&[data-appearance='light'] {
+			background-color: ${neutral[100]};
+		}
+
+		&[data-appearance='medium'] {
+			background-color: ${neutral[60]};
+		}
+
+		&[data-appearance='dark'] {
+			background-color: ${neutral[46]};
+		}
+
+		&[data-bounce='1'] {
+			animation-delay: -0.32s;
+		}
+		&[data-bounce='2'] {
+			animation-delay: -0.16s;
+		}
+		&[data-bounce='3'] {
+			animation-delay: -0.00s;
+		}
```

Comment:
I believe by doing it this way instead of e.g 

```ts
bounce: (appearance, offset) => css`
// ...
`
```

Emotion will only have to parse the generated css once, hence the performance boost. In this case, there will be essentially no difference, but perhaps worth structuring our emotion a little more like this going forward?

Reply:
This is a nice idea, it'd be great to be able to measure any performance gains of this approach vs the function call approach we make now. Like you said it'll probably be minimal in this case, but if we were to adopt it as a convention across the codebase it could have a noticeable difference.